frontend/src/api/apiClient.ts

```diff
@@ -1,20 +1,20 @@
+import { IFusionContext } from '@equinor/fusion'
 import { config } from '../config'
 
+const FUSION_APP_KEY: string = '74b1613f-f22a-451b-a5c3-1c9391e91e68' //Used for FusionContext
+
```

Comment:
Is this key public? Should it not be in a config file or something?

Reply:
It's public yes, but I'll move it to config for readability